server/migrations/20200708183925-subscription-model-upgrade.js

```diff
@@ -3,118 +3,121 @@
 
 module.exports = {
   up: async (queryInterface, DataTypes) => {
-  // add columns
-    await queryInterface.addColumn('Subscriptions', 'chain_id', { type: DataTypes.STRING, allowNull: true });
-    await queryInterface.addColumn('Subscriptions', 'community_id', { type: DataTypes.STRING, allowNull: true });
-    await queryInterface.addColumn('Subscriptions', 'offchain_thread_id', { type: DataTypes.INTEGER, allowNull: true });
-    await queryInterface.addColumn('Subscriptions', 'offchain_comment_id', { type: DataTypes.INTEGER, allowNull: true });
-    await queryInterface.addColumn('Subscriptions', 'chain_event_type_id', { type: DataTypes.STRING, allowNull: true });
-    await queryInterface.addColumn('Subscriptions', 'chain_entity_id', { type: DataTypes.STRING, allowNull: true });
+    return queryInterface.sequelize.transaction(async (t) => {
+    // add columns
+      await queryInterface.addColumn('Subscriptions', 'chain_id', { type: DataTypes.STRING, allowNull: true });
```

Comment:
I think you need to add the `{ transaction: t }` object to each specific action -- reference other migrations for the exact syntax.

Reply:
transactionalize 💪